music/live-coding-ksql-music.adoc

```diff
@@ -136,65 +136,44 @@ Now this persistent query will show in the queries list.
 
 === Create a new table
 
-Next let's work on the `song-feed` topic, which effectively represents a table of songs. Data in a KSQL `TABLE` are required to have key of type `String`. In this demo, the table can be keyed on the song's ID for joins and aggregations to work on that table.
+Next let's work on the `song-feed` topic, which effectively represents a table of songs.
 
-However the original Kafka topic has a key of type `Long` and the ID field is of type `BIGINT`. You can still create a `TABLE` with a few https://docs.confluent.io/current/ksql/docs/syntax-reference.html#what-to-do-if-your-key-is-not-set-or-is-in-a-different-format[simple steps]:
+The original Kafka topic has a key of type `Long`, which maps to ksqlDB's `BIGINT` sql type, and the ID field stores a copy of the key. To register the topic `song-feed`, from the KSQL query editor:
  
-* Create a `STREAM` from the original Kafka topic `song-feed`:
+* Create a `TABLE` from the original Kafka topic `song-feed`:
 
 [source,bash]
 ----
-CREATE STREAM ksql_songfeed WITH (KAFKA_TOPIC='song-feed', VALUE_FORMAT='AVRO');
+CREATE TABLE ksql_song WITH (KAFKA_TOPIC='song-feed', VALUE_FORMAT='AVRO', KEY='ID');
 ----
 
-* View the contents of this stream.  Before running this query, under `Add query properties`, set `auto.offset.reset=Earliest`.
+* View the contents of this table and confirm that the entries in this KSQL table have a `ROWKEY` that matches the String ID of the song.
  
 [source,bash]
 ----
-SELECT * FROM ksql_songfeed EMIT CHANGES limit 5;
+SELECT * FROM ksql_song EMIT CHANGES limit 5;
 ----
- 
-Then `DESCRIBE` the stream to see the fields associated with this topic and notice that the field `ID` is of type `BIGINT`.
+
+Then `DESCRIBE` the table to see the fields associated with this topic and notice that the field `ID` is of type `BIGINT`.
  
 [source,bash]
 ----
-DESCRIBE ksql_songfeed;
+DESCRIBE ksql_song;
 ----
 
-You can also `Describe` the stream via the C3 GUI to see the same output:
+You can also `Describe` the table via the C3 GUI to see the same output:
```

Comment:
note: the image below is now out of date!  It's describing the song _stream_, not the _table_.